### PR TITLE
fix(frontend): Any Sequencer UI support

### PR DESCRIPTION
**Pull request by fenglixa**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**
Address UI display after #399 merged
**Environment tested:**
image

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [X] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

   PR titles examples:
    * `fix(frontend): fixes empty page. Fixes #1234`
       Use `fix` to indicate that this PR fixes a bug.
    * `feat(backend): configurable service account. Fixes #1234, fixes #1235`
       Use `feat` to indicate that this PR adds a new feature. 
    * `chore: set up changelog generation tools`
       Use `chore` to indicate that this PR makes some changes that users don't need to know.
    * `test: fix CI failure. Part of #1234`
        Use `part of` to indicate that a PR is working on an issue, but shouldn't close the issue when merged.

- [x] Do you want this pull request (PR) cherry-picked into the current release branch?
    
    If yes, use one of the following options:
  
    * **(Recommended.)** Ask the PR approver to add the `cherrypick-approved` label to this PR. The release manager adds this PR to the release branch in a batch update.
    *  After this PR is merged, create a cherry-pick PR to add these changes to the release branch. (For more information about creating a cherry-pick PR, see the [Kubeflow Pipelines release guide](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#option--git-cherry-pick).)

### CODE REVIEW

File: frontend/src/lib/StaticGraphParser.ts [hunk 1]
```diff
@@ -106,7 +114,11 @@ function buildTektonDag(graph: dagre.graphlib.Graph, template: any): void {
       task['runAfter'].forEach((depTask: any) => {
         graph.setEdge(depTask, taskName);
       });
-
+    if (anyTasks.includes(task['name'])) {
```
Comment (drewbutlerbb4) [inline frontend/src/lib/StaticGraphParser.ts:117]: Could we add a comment:
`// Adds dependencies for anySequencers from 'anyCondition' annotation`
Or something similar?

File: frontend/src/lib/WorkflowParser.ts [hunk 2]
```diff
@@ -122,7 +131,17 @@ export default class WorkflowParser {
             }
           });
         }
-
+        if (anyTasks.includes(task['name'])) {
```
Comment (drewbutlerbb4) [inline frontend/src/lib/WorkflowParser.ts:134]: And the same comment here

**Review by drewbutlerbb4 — COMMENTED** [review]
Thanks @fenglixa!

Just a couple nits

**Comment by fenglixa** [comment]
@drewbutlerbb4. I addressed your comments, please check. Thanks.

**Pull request merged**